core/payment/examples/debit_note_flow.rs

```diff
@@ -0,0 +1,113 @@
+use bigdecimal::BigDecimal;
+use chrono::Utc;
+use std::time::Duration;
+use ya_client::payment::{PaymentProviderApi, PaymentRequestorApi};
+use ya_client::web::WebClient;
+use ya_client_model::payment::{Acceptance, DocumentStatus, NewAllocation, NewDebitNote};
+
+#[actix_rt::main]
+async fn main() -> anyhow::Result<()> {
+    std::env::set_var("RUST_LOG", "info");
```

Comment:
would be nicer to read it first and default ( or raise ) the level if not to this tests liking

Reply:
It's only a test script, but OK.